src/PowerShellEditorServices/Services/Symbols/Vistors/FindReferencesVisitor.cs

```diff
@@ -125,13 +125,17 @@ public override AstVisitAction VisitCommand(CommandAst commandAst)
         /// <returns>A visit action that continues the search for references</returns>
         public override AstVisitAction VisitFunctionDefinition(FunctionDefinitionAst functionDefinitionAst)
         {
-            (int startColumnNumber, int startLineNumber) = GetStartColumnAndLineNumbersFromAst(functionDefinitionAst);
+            // Get the start column number of the function name,
+            // instead of the the start column of 'function' and create new extent for the functionName
+            int startColumnNumber =
```

Comment:
This is the change that's in master that's being reverted. If you rebase on master this should go away and the tests will (likely) pass.